.github/workflows/ci.yml

```
@@ -95,6 +93,20 @@ jobs:
         # limit our parallelism to half our cores
         run: earthly-ci --no-output +test --hardware_concurrency=64
 
+  yarn-project-test:
+    needs: setup
+    runs-on: ${{ github.actor }}-x86
+    steps:
+      - {uses: actions/checkout@v4, with: { ref: "${{ github.event.pull_request.head.sha }}"}}
+      # Only allow one memory-hunger prover test to use this runner
+      - uses: ./.github/ci-setup-action
+        with:
+          dockerhub_password: "${{ secrets.DOCKERHUB_PASSWORD }}"
+          concurrency_key: yarn-project-test-${{ github.actor }}-x86
+      - name: "Yarn Project Tests"
+        timeout-minutes: 25
+        run: earthly-ci --no-output ./yarn-project/+test
```

```suggestion
        run: earthly --no-output ./yarn-project/+test
```

@ludamad I think we can change this and merge this one now, wdyt?

---

on it